src/sentry/auth/providers/github/views.py

```diff
@@ -33,7 +33,10 @@ def __init__(self, org=None, *args, **kwargs):
     def handle(self, request: Request, helper) -> HttpResponse:
         with GitHubClient(helper.fetch_state("data")["access_token"]) as client:
             if self.org is not None:
+                # if we have a configured org for our oauth provider
                 if not client.is_org_member(self.org["id"]):
+                    # Fetches orgs for the given user
```

Comment:
What is fetching the org for the given user here? I'm not sure I quite understand these comments or maybe their ordering.

Reply:
ah i can make the comments more clear.

yea - `.is_org_member` makes a request to github to fetch the github orgs that the github user is a part of via the access token we received from the oauth callback.

This is separate from the Sentry orgs that the Sentry user is a part of